pkgs/by-name/pu/pushup/package.nix

```
@@ -0,0 +1,35 @@
+{ fetchFromGitHub, buildGoModule, lib, go, makeWrapper }:
+buildGoModule rec {
+  pname = "pushup";
+  version = "0.2";
+
+  src = fetchFromGitHub {
+    owner = "adhocteam";
+    repo = pname;
+    rev = "v${version}";
+    hash = "sha256-9ENXeVON2/Bt8oXnyVw+Vl0bPVPP7iFSyhxwc091ZIs=";
+  };
+
+  vendorHash = null;
+  subPackages = ".";
+  # Pushup doesn't need CGO so disable it.
+  CGO_ENABLED=0;
+  ldflags = [ "-s" "-w" ];
+  nativeBuildInputs = [ makeWrapper ];
+  # The Go compiler is a runtime dependency of Pushup.
+  allowGoReference = true;
+  postInstall = ''
+    wrapProgram $out/bin/${pname} --prefix PATH : ${
```

```suggestion
    wrapProgram $out/bin/${meta.mainProgram} --prefix PATH : ${
```
for completeness sake